### PR TITLE
Personalized Pages with Responsive Design and Dynamic Content

### PR DESCRIPTION
**Pull request by ahmidou09**
**This pull request introduces personalized pages with responsive design and dynamic content to the project.** 

- The changes made adhere to the general requirements, including the absence of linter errors and correct GitHub flow.

- The HTML/CSS and JavaScript requirements have been fulfilled by following the best practices for each. 

- The pages now incorporate personalized content and adhere to the design guidelines provided, encompassing colors, typography (utilizing the suggested Lato font), and layout composition with appropriate spacing.

- Two pages have been built: the home page and the about page, each with versions tailored for mobile (up to 768px wide) and desktop (768px or wider) screens. Navigation has been improved with a link from the home page menu to the about page, and the logo in the header now links back to the home page.

- A responsive mobile menu has been implemented, triggered by the hamburger button, which overlays the page. The mobile menu design aligns with the overall design, including colors, typography, and spacing. Additionally, a close (X) button has been added to allow users to easily close the mobile menu.

- Furthermore, the "Professional Expertises" section has been dynamically created using JavaScript. A JavaScript variable containing Professional Expertises data is now utilized upon page load to generate the HTML dynamically, ensuring a dynamic and engaging user experience.

 [video presenting the features of the project  ](https://www.loom.com/share/a6ff655f6d6d4fd09c1bede154b0f50a)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Thanks to my learning partners and the staff at Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (mukhammadrizooff) [inline README.md]: - [ ] [REQUIRED] Good job so far! The `README` contains an `"Acknowledgements"` section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Please implement my suggestion 🙏🏼 

image

- Kindly this part should be placed inside of `"Acknowledgements"` section 👍🏼 

image

File: README.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```
Comment (mukhammadrizooff) [inline README.md]: - [ ] [REQUIRED] Good job so far! Kindly your `LICENSE` is not working, please change the name of file to correct one(`MIT.md`)🙏🏼 

image

File: index.html [hunk 3]
```diff
@@ -83,13 +83,10 @@
         <div class="headline">
```
Comment (mukhammadrizooff) [inline index.html:83]: - [ ] [REQUIRED] Good job so far! I appreciate what you have done I really appreciate what you have done so far but `some elements`("date" part) on this page are missing, please style them like [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

- `[HINT]` The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design.

  * Basic color swatch: `#272A31`, `#EC5242`, `#D3D3D3`

- The pages use the same fonts or simialr fonts from the original design.

  * Fonts used originally: `Cocogoose, Lato`

- All page elements have similar spacing, borders, shadows, and paddings than the orignal design.

- `[DESKTOP]`
image

- `[MOBILE]`
image

File: index.html [hunk 4]
```diff
@@ -104,7 +101,7 @@ <h2 class="heading-secondary">beautiful shortcode</h2>
             <h3
               class="heading-tertiary c-light u-center-text u-margin-bottom-small"
             >
-              Main Program
+              Best Popular Services
```
Comment (mukhammadrizooff) [inline index.html:108]: - [ ] [REQUIRED] Good job so far! Please give correct design to button of `"SEE THE WHOLE PROGRAM"` for desktop, unfortunately you give `mobile` design for all sides(`desktop` and `mobile`) 🙏🏼 

image

File: index.html [hunk 5]
```diff
@@ -393,10 +229,7 @@ <h3 class="heading-tertiary u-center-text u-margin-bottom-small">
         </div>
       </div>
       <p class="copy">
-        2015 Creative Commons Korea. Some Rights Reserved. <br />
-        달리 정하지 않은 웹사이트의 모든 내용은 크리에이티브 커먼즈 저작자표시
-        2.0 대한민국 라이선스에 따라 이용할 수 있습니다.<br />
-        http://creativecommons.org/licences/by/2.0/kr
+        2015 Creative Commons Korea. SALL RIGHTS RESERVED..
```
Comment (mukhammadrizooff) [inline index.html]: - [ ] [REQUIRED] Good job so far! Please decrese the size of space between `Footer Texts` as like [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

image

File: page/about.html [hunk 6]
```diff
@@ -22,7 +22,7 @@
         <div class="container">
```
Comment (mukhammadrizooff) [inline page/about.html:22]: - [ ] [REQUIRED] Good job so far! I appreciate what you have done I really appreciate what you have done so far but `main texts` need to correct style, please style them like [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

- `[HINT]` The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design.

  * Basic color swatch: `#272A31`, `#EC5242`, `#D3D3D3`

- The pages use the same fonts or simialr fonts from the original design.

  * Fonts used originally: `Cocogoose, Lato`

- All page elements have similar spacing, borders, shadows, and paddings than the orignal design.

- `[MOBILE]`
image

File: page/about.html [hunk 7]
```diff
@@ -106,11 +107,10 @@ <h2 class="heading-secondary u-center-text u-margin-bottom-medium">
...
         <div class="row flex-d-culumn">
           <h3 class="heading-tertiary u-center-text u-margin-bottom-medium">
-            See the past CC Global Summits
+            Come closer to our web agency
           </h3>
           <h2 class="heading-secondary u-center-text u-margin-bottom-medium">
-            Take a look at the last two CC Global Summits which took place in
-            Buenos Aires and in Warsaw.
+            website creation: from an experienced web agency
           </h2>
```
Comment (mukhammadrizooff) [inline page/about.html:117]: - [ ] [REQUIRED] Good job so far! Please remove the `footer` part from `About` page and pay attention to your `text designs` 🙏🏼 

image

File: main.js
Comment (mukhammadrizooff) [inline main.js:1]: - [ ] [REQUIRED] Good job so far! Please give correct style to `LESS`&`MORE` buttons as like [Behance ](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @ahmidou09, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitHub Flow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Efiamotu-1 — APPROVED** [review]
Hi @ahmidou09 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

giphy


## Highlights :dart: 
* Correct GitHub flow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PR Title & summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**